content/en/docs/reference/glossary/namespace-sameness.md

```diff
@@ -5,7 +5,7 @@ test: n/a
 
 Within a multicluster mesh, [namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md)
 applies and all namespaces with a given name are considered to be the same namespace. If multiple clusters contain a
-`Service` with the same namespace name, they will be recognized as a single combined service. By default, traffic is
+`Service` with the same namespaced name, they will be recognized as a single combined service. By default, traffic is
```

Comment:
See my comments in the original PR. I'm not sure that there is a good definition for `namespaced`.
```suggestion
`Service` with the same name in a same named namespace, they will be recognized as a single combined service. By default, traffic is
```